controllers/apply/awards-for-all/constants.js

```diff
@@ -109,6 +109,12 @@ const FILE_LIMITS = {
     ]
 };
 
+const APPLICATION_PROGRESS_STATES = {
```

Comment:
I don't think this should be defined here, but rather set as a ENUM / field limit on the database.